### PR TITLE
feat(button): add transparent button

### PR DESCRIPTION
**Pull request by tverskih**
Transparent buttons added for light and dark themes.

image

image

Reviewers:
@Fost @lskramarov @mikeozornin

### CODE REVIEW

**Comment by mikeozornin** [comment]
Есть совсем прямо мелочь-мелочь.

Progress + disabled, progress + normal, progress + pressed
В этих состояниях карамелька (двигающийся прогресс) без бордер-радиуса. Когда ховер или фокус, то все ок.

Слева hover, справа pressed:
image

**Comment by tverskih** [comment]
I've updated border radius to be inherited from the parent.
@lskramarov please take a look.

File: packages/mosaic/core/common-behaviors/color.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export enum ThemePalette {
     Second = 'second',
     Error = 'error',
     Default = 'second',
+    Transparent = 'transparent',
```
Comment (lskramarov) [inline packages/mosaic/core/common-behaviors/color.ts]: it is bad idea, we dont have palette `transparent`.

I dont know how do it better, we will discuss it today.

Comment (Fost) [reply packages/mosaic/core/common-behaviors/color.ts]: good point

Comment (tverskih) [reply packages/mosaic/core/common-behaviors/color.ts]: @lskramarov, as an option: 
- set this button with secondary color scheme (beacause it's closer by style to secondary) and add a class "transparent": `<button color="secondary" class="transparent">`

Also let's figure out - is button control - the only tranparent control for now? If yes, then this solution seems to be reasonable.

Comment (lskramarov) [reply packages/mosaic/core/common-behaviors/color.ts]: yes, we had discussed it and decided do it with class modificator `.mc-button_transparent`, this class should work with any color.

Comment (tverskih) [reply packages/mosaic/core/common-behaviors/color.ts]: @lskramarov updated with new approach.

**Pull request merged**